articles/css3-text/index.en.html

```diff
@@ -290,7 +290,7 @@ <h2>You can help</h2>
   <p>You can help move the specifications forward by reviewing and commenting on the public drafts that are made available. If you have
       expertise in this area, you might also consider participating in the CSS or Internationalization Working Groups to help move the work along. </p>
     <p>A
-      significant amount of specification work benefitted from the <a href="https://www.w3.org/TR/jlreq/">Japanese Layout Requirements</a> document, and work is going on to <a href="https://www.w3.org/International/layout">document requirements for other scripts</a>. You can contribute to that work, if you have expertise in that area.</p>
+      significant amount of specification work benefitted from the <a href="https://www.w3.org/TR/jlreq/">Japanese Layout Requirements</a> document, and work is going on to <a href="https://www.w3.org/International/i18n-drafts/nav/languagedev">document requirements for other scripts</a>. You can contribute to that work, if you have expertise in that area.</p>
```

Comment:
The URL seems less good than the old one? Why would the drafts version be pointed to vs. the old location?